schedulers/terraform/managed-airflow-mwaa/data.tf

```diff
@@ -1,5 +1,5 @@
 data "aws_eks_cluster_auth" "this" {
-  name = module.eks_blueprints.eks_cluster_id
+  name = module.eks.cluster_name
 }
 
```

Comment:
This policy data "aws_iam_policy_document" "emr_on_eks"  might not be required with the new emr_containers module